test/unit/processing/calculation/get-calculation.test.js

```diff
@@ -47,114 +47,114 @@ describe('get and transform payment request information for building a statement
     jest.clearAllMocks()
   })
 
-  test('should call getCalculationByPaymentRequestId when a paymentRequestId is given', async () => {
-    const paymentRequestId = 1
-    await getCalculation(paymentRequestId, mockTransaction)
+  test('should call getCalculationByPaymentRequestId when a paymentRequest is given', async () => {
+    const paymentRequest = { paymentRequestId: 1 }

```

Comment:
Let's use `mock-payment-request` obj

Reply:
moved to https://eaflood.atlassian.net/browse/PAY-2364